runner/master/run.sh

```diff
@@ -385,7 +424,7 @@ then
       --network "${NETWORK}" \
       --name ${IONICHOSTNAME} \
       --detach \
-      moodlehq/moodlemobile2:latest
+      moodlehq/moodlemobile2:"$MOBILE_VERSION"
```

Comment:
Now that we've decided to require the plugin in order to run the `@app` tests, I'd suggest we also require it before running the mobile2 image.

Reply:
Ah... Isee, we may not be able to do so because we've gone for the generic 'PLUGINS' rather than specific.
In which case, maybe we should have the default for `MOBILE_VERSION` as an empty string, and only start the image if a version is specified?